### PR TITLE
TS-refactor-styles-in-semester-and-rooms-pages

### PR DESCRIPTION
**Pull request by ViktoriiaKosovan**

### CODE REVIEW

File: frontend/src/App.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-@import "./styles/fonts.scss";
-@import "./styles/variables.scss";
-@import "./styles/mixins.scss";
+@import './styles/colors.scss';
+@import './styles/fonts.scss';
+@import './styles/variables.scss';
+@import './styles/mixins.scss';
 
 html {
     height: 100%;
```
Comment (Niknikson) [inline frontend/src/App.scss:7]: use $full-size

File: frontend/src/App.scss [hunk 2]
```diff
@@ -47,7 +47,7 @@ body {
     position: relative;
 
     &::after {
-        content: "";
+        content: '';
         position: absolute;
         left: 0;
         width: 100%;
```
Comment (Niknikson) [inline frontend/src/App.scss:53]: use $full-size

File: frontend/src/components/RoomsPage/RoomTypeForm/RoomTypeForm.scss [hunk 3]
```diff
@@ -9,43 +16,28 @@
     margin: 15px 0;
 
     .type-form-btn {
-        width: 96%;
+        width: $w-full;
```
Comment (Niknikson) [inline frontend/src/components/RoomsPage/RoomTypeForm/RoomTypeForm.scss]: use $full-size

File: frontend/src/components/RoomsPage/RoomsList/RoomsList.js [hunk 4]
```diff
@@ -26,9 +26,9 @@ const RoomList = (props) => {
         return <NotFound name={t(ROOM_Y_LABEL)} />;
     }
     return (
-        <section className="container-flex-wrap wrapper">
+        <section className="container-flex-wrap">
             {visibleItems.map((room) => (
-                <RoomCard key={room.id} room={room} isDisabled={isDisabled} {...rest} />
+                <RoomCard key={room.id} {...rest} room={room} isDisabled={isDisabled} />
```
Comment (Niknikson) [inline frontend/src/components/RoomsPage/RoomsList/RoomsList.js:31]: use {...props}

File: frontend/src/components/Semester/SemesterForm/SemesterForm.js [hunk 5]
```diff
@@ -141,7 +140,7 @@ const SemesterForm = (props) => {
                 onClose={closeDialogForGroup}
             />
             <form onSubmit={handleSubmit}>
-                <div className="semester-checkbox">
+                <div className="semester-checkboxes">
                     <div>
```
Comment (Niknikson) [inline frontend/src/components/Semester/SemesterForm/SemesterForm.js:144]: do you need this empty div ?

File: frontend/src/components/Semester/SemesterForm/SemesterForm.scss [hunk 6]
```diff
@@ -1,44 +1,32 @@
...
-        justify-content: space-between;
-        margin: 5px 0;
-        .semester-field-input {
-            width: 45%;
-        }
+.semester-inputs-block {
+    @include displayFlex(center, space-between);
+    margin: 5px 0;
+    .semester-field-input {
+        width: $w-50;
```
Comment (Niknikson) [inline frontend/src/components/Semester/SemesterForm/SemesterForm.scss]: $w-50

File: frontend/src/App.scss (hunk 1 above)
Comment (ViktoriiaKosovan) [reply frontend/src/App.scss:7]: it will refactor in next pull

File: frontend/src/App.scss (hunk 2 above)
Comment (ViktoriiaKosovan) [reply frontend/src/App.scss:53]: it will refactor in next pull

File: frontend/src/components/Semester/SemesterForm/SemesterForm.js (hunk 5 above)
Comment (ViktoriiaKosovan) [reply frontend/src/components/Semester/SemesterForm/SemesterForm.js:144]: yes, I need it for styling fields in one row in centre

File: frontend/src/components/RoomsPage/RoomTypeForm/RoomTypeForm.scss (hunk 3 above)
Comment (ViktoriiaKosovan) [reply frontend/src/components/RoomsPage/RoomTypeForm/RoomTypeForm.scss]: done

File: frontend/src/components/Semester/SemesterForm/SemesterForm.scss (hunk 6 above)
Comment (ViktoriiaKosovan) [reply frontend/src/components/Semester/SemesterForm/SemesterForm.scss]: done

File: frontend/src/components/RoomsPage/RoomTypeForm/RoomTypeForm.scss (hunk 3 above)
Comment (ViktoriiaKosovan) [reply frontend/src/components/RoomsPage/RoomTypeForm/RoomTypeForm.scss]: done

File: frontend/src/components/Semester/SemesterForm/SemesterForm.scss (hunk 6 above)
Comment (ViktoriiaKosovan) [reply frontend/src/components/Semester/SemesterForm/SemesterForm.scss]: done

File: frontend/src/components/RoomsPage/RoomsList/RoomsList.js (hunk 4 above)
Comment (ViktoriiaKosovan) [reply frontend/src/components/RoomsPage/RoomsList/RoomsList.js:31]: i cant use props, because props is already declared as parameter of function

**Review by HiroFive — APPROVED** [review]

**Review by Niknikson — APPROVED** [review]

**Pull request merged**